roles/vault_pki/tasks/prechecks.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Fail if variables are not set"
+  fail:
+    msg: "variable {{ item }} is not set"
+  when:
+    - vars[item.name] | length == 0
```

Comment:
Is `vars` a new thing? I haven't seen it before (only `hostvars`). Does it require a minimum version of ansible?

Reply:
I probably found that somewhere over the internet, seems to work on Ansible 2.10 as well - do we want to move to hostvars[item.name] instead?

Reply:
hostvars has some issues in that it only picks up certain variable sources. If vars fixes that then it would be better. I guess if it passes CI it should be ok.